### PR TITLE
Account refactor

### PR DESCRIPTION
**Pull request by Suyashtnt**
accounts are now an array
the vuex store is refactored
the vuex store is also saved to disc
you can select any account at any time
dialog made to add accounts
tailwind is removed for bulma

NOTE: the add user button is temporary and will be in the list selection once its added to primevue(primefaces/primevue#833)

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -14,17 +14,75 @@
...
 			</template>
+			<template #right>
+				<Dropdown
+					v-model="selectedUser"
+					:options="users"
+					optionLabel="name"
+					placeholder="no user selected"
+					class="mr-3"
+				/>
+				<Button @click="userAddPopup = true">add user</Button>
```
Comment (Milo123459) [inline src/App.vue:25]: Add User would look better imo

Comment (Suyashtnt) [reply src/App.vue:25]: shall change

File: src/store/state.ts [hunk 2]
```diff
@@ -3,9 +3,9 @@ import { IUser } from 'minecraft-launcher-core';
 import persistentStore from './persistant';
 
 export const state = {
-	sidebar: false as boolean,
 	instances: (persistentStore.get('instances') as Schema[]) || ([] as Schema[]),
-	user: (persistentStore.get('user') as IUser) || ({} as IUser),
+	users: (persistentStore.get('users') as IUser[]) || ([] as IUser[]),
```
Comment (Milo123459) [inline src/store/state.ts:7]: use a union type here

**Review by Milo123459 — APPROVED** [review]

File: src/store/state.ts (hunk 2 above)
Comment (Suyashtnt) [reply src/store/state.ts:7]: ?

Comment (Milo123459) [reply src/store/state.ts:7]: Array<IUser> is a union type

Comment (Milo123459) [reply src/store/state.ts:7]: Just means adding the <>, ie, Array<IUser>

**Review by Milo123459 — APPROVED** [review]

File: src/store/state.ts (hunk 2 above)
Comment (Suyashtnt) [reply src/store/state.ts:7]: sorry but generics are not allowed

Comment (Suyashtnt) [reply src/store/state.ts:7]: eslint config

Comment (Milo123459) [reply src/store/state.ts:7]: Why not?

Comment (Suyashtnt) [reply src/store/state.ts:7]: looks ugly to me

**Pull request merged**